### PR TITLE
Push icon #119

### PR DESCRIPTION
**Pull request by soulgalore**
Show an icon if the response is server pushed.
screen shot 2017-01-16 at 21 05 17

### CODE REVIEW

**Comment by soulgalore** [comment]
Ooops will fix that tomorrow.

File: src/ts/helpers/heuristics.ts [hunk 1]
```diff
@@ -69,6 +69,13 @@ export function isSecure(entry: WaterfallEntry) {
   return entry.name.indexOf("https://") === 0;
 }
 
+export function isPush(entry: WaterfallEntry) {
+  const harEntry = entry.rawResource;
+  // WebPageTest got 1 if an asset was pushed, Browsertime = true
+  console.log(harEntry["_was_pushed"]);
```
Comment (tobli) [inline src/ts/helpers/heuristics.ts]: Drop the console.log here.

File: src/ts/helpers/heuristics.ts [hunk 2]
```diff
@@ -69,6 +69,13 @@ export function isSecure(entry: WaterfallEntry) {
...
+  return ( harEntry["_was_pushed"] === true || Number(harEntry["_was_pushed"]) === 1) ? true : false;
```
Comment (tobli) [inline src/ts/helpers/heuristics.ts]: The conditional (?:) isn't needed here. I might even go with ```!!harEntry["_was_pushed"]```. Does WPT put a number (1) or a string ("1")? The Number conversion looks a bit strange.

Comment (soulgalore) [reply src/ts/helpers/heuristics.ts]: WPT uses a String "1".

File: src/ts/helpers/heuristics.ts (hunk 1 above)
Comment (soulgalore) [reply src/ts/helpers/heuristics.ts]: Will remove that.

**Review by micmro — APPROVED** [review]

File: src/ts/helpers/heuristics.ts (hunk 2 above)
Comment (tobli) [reply src/ts/helpers/heuristics.ts]: Aha, so maybe explicit check against "1" would be clearer. But very minor nitpick.

**Comment by soulgalore** [comment]
@tobli do you want to have a go to push the sign to the just before the bar? I also like that, don't know how to fix it though.

**Comment by tobli** [comment]
Let's merge this first, I can try putting the arrow on the left later.

**Comment by soulgalore** [comment]
or we keep the "1" as a protest.

**Pull request merged**